habitat_sim/agent/controls/__init__.py

```diff
@@ -8,6 +8,7 @@
     ActuationSpec,
     ObjectControls,
     SceneNodeControl,
+    move_func_map,
```

Comment:
This map is really more of an internal thing.  You shouldn't interact with it directly IMO.  You can if you need to, but I don't think it should be officially in the public API.

Reply:
It's referenced from here, which triggered me to expose it: https://tmp.mosra.cz/habitat-sim/habitat_sim.agent.ObjectControls.html#action

Reply:
Ah.  Makes sense then.  For the doc for https://tmp.mosra.cz/habitat-sim/habitat_sim.agent.html#move_func_map, I am not 100% sure how that is generated, but it looks like it is just a print.  You can try https://docs.python.org/3/library/pprint.html for a much nicer looking print :)

Reply:
Right now it's just what `str(thing)` returns, yes, as that worked well for basic constants and enums (see e.g. https://doc.magnum.graphics/python/magnum.math.html#data). This is on my TODO list, I'm thinking about completely omitting the value if it's too long -- for long dicts and lists (also https://tmp.mosra.cz/habitat-sim/habitat_sim.utils.html#data) the listing doesn't add much value anyway. If the contents are really important I think there should be a detailed doc block for it with a human-readable list instead. (Or the relevant snippet included as a highlighted code block, at least -- easy with the `.. include::` directive.)

Btw., this is exactly the feedback I'm looking for, so thanks a lot! :+1: